bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/gen3/dto/SceneEvent3.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hdpowerview.internal.gen3.dto;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Scene SSE event object as supplied an HD PowerView hub of Generation 3.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@NonNullByDefault
+public class SceneEvent3 {
+    private int id;
+    // private @NonNullByDefault({}) String evt;
+    // private @NonNullByDefault({}) String isoDate;
```

Comment:
Can this be removed?

Reply:
I think we could certainly delete the 'isoDate' field as I can see no potential purpose for it in OH.

By contrast the 'evt' field might or might not be useful -- it depends on the testing -- this field contains a message like 'motion-started' or 'motion-completed', which we currently do not need, since for both such events we consume the 'current position' from the respective event. However testing might potentially show that 'evt' might potentially have some relevance in reality. So I suggest to leave it the _ad interim_.